content-test/common/reducers/equality.test.js

```diff
@@ -0,0 +1,35 @@
+const createStore = require("common/create-store");
+const {actions} = require("common/action-manager");
+const {createRows} = require("test/faker");
+
+describe("Reducer equality", () => {
```

Comment:
There's something incongruent about this title.  All of the "it" clauses are not specifically referring to "Reducer equality", I don't think.  Perhaps this doesn't matter; perhaps there's another phrasing that would improve readability.